### ISSUE TITLE
diceroll modifier can't be less then 0

### ISSUE THREAD

**Issue by Aer0nic** [Redjumpman/Jumper-Cogs#48]
Most gaming systems make use of negative modifiers, so why doesn't this allow for that?

```
[13/04/2017 15:50] ERROR red on_command_error 375: Exception in command 'dtable roll'
Traceback (most recent call last):
  File "lib\discord\ext\commands\core.py", line 50, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "n:\Red-DiscordBot\cogs\dicetable.py", line 58, in _roll_dtable
    raise NegativeValue("Modifier can't be less than 0.")
cogs.dicetable.NegativeValue: Modifier can't be less than 0.
```

Using current versions of RedDiscord and Diceroll

**Comment by Redjumpman** [comment]
Yikes. That's a mistake. The original intent for the negative value was supposed to be for people trying to roll a negative number of die. I'll try to get a fix out for it when I can. Thanks for the report.

**Comment by Redjumpman** [comment]
Fixed in Patch 2.01 commit efe62c6
